listenbrainz/db/model/recommendation_feedback.py

```diff
@@ -0,0 +1,42 @@
+import uuid
+
+from datetime import datetime
+from pydantic import BaseModel, ValidationError, validator
+
+
+class RecommendationFeedback(BaseModel):
+    """ Represents a recommendation feedback object
+        Args:
+            user_id: the row id of the user in the DB
+            user_name: (Optional) the MusicBrainz ID of the user
+            recording_mbid: the MusicBrainz ID of the recording
+            feedback: the feedback associated with the recording
+            created: (Optional)the timestamp when the feedback record was inserted into DB
+    """
+
+    user_id: int
+    user_name: str = None
+    recording_mbid: str
+    feedback: str
+    created: datetime = None
+
+    @validator('feedback')
+    def check_feedback_is_valid(cls, feedback):
+        expected_feedback = [
+            'I never want to hear this again',
```

Comment:
I guess expected_feedback should be same as enum https://github.com/metabrainz/listenbrainz-server/blob/rec-component-feedback/admin/sql/create_types.sql#L7